tests/MongoDB.Driver.Core.Tests/Specifications/server-discovery-and-monitoring/tests/rs/repeated.yml

```diff
@@ -0,0 +1,101 @@
+description: Repeated ismaster response must be processed
+
+uri: "mongodb://a,b/?replicaSet=rs"
+
+phases:
+  # Phase 1 - a says it's not primary and suggests c may be the primary
```

Comment:
this file contains the test which led us to the last commit in this PR with this change: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/83/files#diff-346021f0b586b815ec28dc10028a0a5aR321-R322